src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -824,10 +824,10 @@ form of flowcharts.
 Data acquired to correct for *B<sub>0</sub>* inhomogeneities can come in different forms.
 The current version of this standard considers four different scenarios:
 
- 1.  [Phase-difference map](#case-1-phase-difference-map-and-at-least-one-magnitude-image)
- 1.  [Two phase maps](#case-2-two-phase-maps-and-two-magnitude-images)
- 1.  [Direct *field mapping*](#case-3-direct-field-mapping)
- 1.  ["*PEpolar*" fieldmaps](#case-4-multiple-phase-encoded-directions-pepolar)
+1.  [Phase-difference map](#case-1-phase-difference-map-and-at-least-one-magnitude-image)
+1.  [Two phase maps](#case-2-two-phase-maps-and-two-magnitude-images)
+1.  [Direct *field mapping*](#case-3-direct-field-mapping)
+1.  ["*PEpolar*" fieldmaps](#case-4-multiple-phase-encoded-directions-pepolar)
```

Comment:
my changes apparently switched some checks "back on" ... this is a valid issue that was not raising an alarm recently (but used to do so some time ago).